tests/Dfc.CourseDirectory.Testing/DataStore/CosmosDb/Queries/CreateCourse.cs

```diff
@@ -47,5 +47,6 @@ public class CreateCourseCourseRun
         public CourseAttendancePattern AttendancePattern { get; set; }
         public bool? National { get; set; }
         public IEnumerable<string> Regions { get; set; }
+        public string ProviderCourseId { get; set; }
```

Comment:
It's not immediately clear to me why this change was needed from the code or the commit message. Suggest amending the relevant commit message to add reason for needing to extend this.
I could see it was something test related, but not why the tests now needed this when they didn't before.